server/controllers/stock/reports/index.js

```diff
@@ -592,9 +592,29 @@ function stockLotsReport(req, res, next) {
   let options = {};
   let display = {};
   let hasFilter = false;
+  let filters = [];
 
   const data = {};
   let report;
+
+// getting params
+  var defaultParam = req.query;
+  try {
+
+    const keys = Object.keys(defaultParam);
+    keys.forEach((key) => {
+      if ((key !== 'lang') && (key !== 'renderer')) {
+        filters.push({ displayName : key, value : defaultParam[key] });
+      }
+    }, keys);
+    // filters paramter found
+    hasFilter = filters.length > 0;
+
+  } catch (error) {
```

Comment:
Errors should almost always be handled in some way in production.  There has to be a really good reason to have a blank `catch`.  At minimum, we should log something using `winston.error()` to let the application developers know that something when wrong.